CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# dbt_netsuite v0.14.0
```

Comment:
Please also include the relevant upstream breaking and other important changes in this changelog entry

Reply:
Added.